tests/test_openjdk.py

```diff
@@ -71,7 +71,11 @@ def test_jdk_version(container, java_version):
         (
             "threads_concurrency_and_sleep",
             ["I am the thread 1", "I am the thread 2"],
-        )
+        ),
+        (
+            "files_and_directories",
+            ["ALL OK"],
+        ),
     ],
 )
 def test_jdk_extended(
```

Comment:
Please update the docstring of this function.

Reply:
done